docs/book/driver-deployment/prerequisites.md

```diff
@@ -0,0 +1,156 @@
+| CNS-VM                  | ![ROLE-CNS-VM](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/ROLE-CNS-VM.png)<br><pre lang="bash">% govc role.ls CNS-VM<br>System.Anonymous<br>System.Read<br>System.View<br>VirtualMachine.Config.AddExistingDisk<br>VirtualMachine.Config.AddRemoveDevice<br></pre>| All node VMs                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| CNS-SEARCH-AND-SPBM     | ![ROLE-CNS-SEARCH-AND-SPBM](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/ROLE-CNS-SEARCH-AND-SPBM.png)<br><pre lang="bash">% govc role.ls CNS-SEARCH-AND-SPBM<br>Cns.Searchable<br>StorageProfile.View<br>System.Anonymous<br>System.Read<br>System.View<br></pre>| Root vCenter Server                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| ReadOnly                | This role is already available in the vCenter.<br><pre lang="bash">% govc role.ls ReadOnly<br>System.Anonymous<br>System.Read<br></pre>| Users with the Read-Only role for an object are allowed to view the state of the object and details about the object. <br>For example, users, with this role can find the shared datastore accessible to all node VMs. <br><br>For, zone and topology-aware environments, all ancestors of node VMs, such, as a host, cluster, and data center must have the Read-only role set for, the vSphere user configured to use the CSI driver and CPI.<br>This is required to allow reading tags and categories to prepare the nodes' topology. |
```

Comment:
Remove comma:
s/For, zone/For zone

Reply:
Remove comma after "node VMs" - s/all ancestors of node VMs, /all ancestors of node VMs

Remove comma after "such" - s/such, as a host/such as a host

Reply:
done

Reply:
Remove comma in the middle:

s/the Read-only role set for, the vSphere/the Read-only role set for the vSphere

Reply:
done. not sure why so many unwatend `,` added. 
thanks for catching them.